pkg/volume/util/fsquota/quota.go

```diff
@@ -44,6 +44,6 @@ type Interface interface {
 	ClearQuota(m mount.Interface, path string) error
 }
 
-func enabledQuotasForMonitoring() bool {
+func EnabledQuotasForMonitoring() bool {
```

Comment:
Why did you export this?

Reply:
staticcheck reported this function never used. However, `/pkg/volume/util/fsquota/quota_linux.go` is calling this function (same package scope)